packages/web3-shared/base/src/utils/resolver.ts

```diff
@@ -443,6 +443,10 @@ export function resolveArweaveURL(url: string | undefined) {
     return urlcat('https://arweave.net/:str', { str: url })
 }
 
+/**
+ * Please do not use to resolve an image or an video resource, because that's
+ * not allowed by the cors agent server
```

Comment:
can you explain more clearly?

Reply:
Our CORS agent server just blocked image request few days ago.

Reply:
Image proxy consume lots of traffic, costs too much money.